spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -566,6 +566,9 @@ If no exception is thrown by observers then the resulting CompletionStage is com
 
 Before the actual observer notification, the container determines an order in which the observer methods for a certain event are invoked.
 The priority of an observer method may be declared using the `@Priority` annotation.
```

Comment:
The priority of an observer method may be declared **by annotating the event parameter with `@Priority` annotation**.

Otherwise looks good ;-)